gstd/src/common/handlers.rs

```diff
@@ -25,43 +25,47 @@
 //! debug and non-debug mode, for programs built in `wasm32` architecture.
 //! For `debug` mode it provides more extensive logging.
 
-#[cfg(target_arch = "wasm32")]
-use {crate::ext, core::alloc::Layout, core::panic::PanicInfo};
+#[cfg(feature = "oom-handler")]
+use crate::ext;
```

Comment:
For the case you can move it inside function to make file more readable

Reply:
Guess it's better to remove the `use` altogether then

Reply:
Fixed in 2489cd2.